sandbox/grommet-app/src/pages/layouts/kinds/RecordDetail/DetailSummary.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Box, NameValueList, NameValuePair, Text } from 'grommet';
+import { StatusGoodSmall } from 'grommet-icons';
+import { sentenceCase } from '../../../../utils/format';
+
+const groupDetails = {
+  status: {
+    value: 'Okay',
+    render: () => <Box direction="row" gap="xsmall" align="center">
+      <StatusGoodSmall color='status-ok' />
+      <Text>Okay</Text>
+    </Box>
+    ,
+  },
+  state: 'Job in progress',
+  group: 'Server group name',
+  servers: 2,
+  baseline: 'SPP 2022.09.04(04 Sep 2022)',
+  power: 'On',
+};
+
+const nameProps = {
+  width: ['xsmall', 'max-content']
+};
+
+export const DetailSummary: React.FC = () => {
+  return (
+    <NameValueList nameProps={nameProps}>
```

Comment:
locally Im getting a TS error for the `nameProps` I wanted to check on your side if it was fine or getting same error.

Reply:
Yes, I get the error too. I'm leaving it because I think Grommet should update the width type definition for NameValueList as it using Grid underneath and an array with min/max values should be valid. Leaving the error as a reminder that this is something that should be fixed.

Reply:
agree I was thinking grommet should be enhanced!

Reply:
@halocline Can you file a ticket on this in Grommet?

Reply:
https://github.com/grommet/grommet/pull/7509

one step ahead I put up a PR on this

Reply:
sorry thought I had commented that I will put up a PR